docs/new-architecture-backward-turbomodule.md

```diff
@@ -0,0 +1,699 @@
+
+If you are interested in creating a Turbomodule, you can skip this part and directly go [here](#create-the-turbomodule). Consider that most of the code of a Native Module is needed for a Turbomodule.
+
+### Creating the Structure
+
+We have two alternatives to create the basic structure for our module.
+
+1. Using the React Native CLI.
+2. Manually.
+
+#### React Native CLI
+
+To create a turbomodule with the React Native CLI, run the following command:
+
+```sh
+npx create-react-native-library <name-of-the-library>
+```
+
+Follow the questions the CLI asks and, once the process is done, it will create a folder structure completed with an example app, typescript configuration and much more. The sample structure is shown in the following image:
+
+<figure>
+  <img src="docs/assets/Turbomodules/NpxLibFolderStructure.png" />
+  <figcaption>The folder structure generated by the React Native CLI</figcaption>
+</figure>
+
+This is a good starting point, but remember to double check what will be published if you keep this setup. Moreover, the `TypeScript` support for the new architecture is still in beta, so you won't be able to use it for the Turbomodule sections.
+
+#### Manually
+
+The second approach is to create a similar structure manually. What we need to start is a folder named `example-library` and a `package.json` file with the following content:
+
+```json title="package.json"
+{
+  "name": "example-library",
+  "version": "0.0.1",
+  "description": "Showcase Turbomodule with Backcompatibility ",
+  "react-native": "src/index",
+  "source": "src/index",
+  "files": [
+    "src",
+    "android",
+    "ios",
+    "example-library.podspec",
+    "!android/build",
+    "!ios/build",
+    "!**/__tests__",
+    "!**/__fixtures__",
+    "!**/__mocks__"
+  ],
+  "keywords": ["react-native", "ios", "android"],
+  "repository": "https://github.com/<your_github_handle>/example-library",
+  "author": "<Your Name> <your_email@your_provide.com> (https://github.com/<your_github_handle>)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/<your_github_handle>/example-library/issues"
+  },
+  "homepage": "https://github.com/<your_github_handle>/example-library#readme",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/"
+  },
+  "devDependencies": {},
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*"
+  }
+}
+```
+
+For sake of simplicity, we will go on with this manual setup, adding files while we work.
+Remember to update the `<Your Name>`, `<your_email@your_provide.com>`, and all the `<your_github_handle>` tokens, if you want to follow this approach with your information.
+
+### Create a Testing Environment
+
+To develop a Native Module we would like to test the JS side and to develop safely the native parts. A library usually does not come with a testing aplication out-of-the-box. So, we are going to need one.
```

Comment:
> we would like to test the JS side and to develop safely the native parts

I'm not sure I follow here. Why you want to test ONLY the JS side. And what does "develop safely" means in this context?